### ISSUE TITLE
cannot expand/collapse Notes List with keyboard

### ISSUE THREAD

**Issue by dwhieb** [digitallinguistics/app#330]
There's no way to expand/collapse a Notes List using the keyboard.

This will probably require making the expand/collapse icon a button.

Maybe the entire Notes List `<header>` could be a `<button>` instead. Hmm...
